docs/docker-example/Dockerfile

```diff
@@ -10,19 +10,18 @@ RUN apt-get install -y python3-pip
 RUN python -m pip install --upgrade pip
 RUN apt-get install -y gdb
 RUN apt-get install -y lldb
-RUN apt-get install -y emacs
```

Comment:
Curious: is vim really required ?  (Asking because I see emacs has been removed buf vim has not)

Reply:
ok, put both as optional... actually, its unclear also that lldb and gdb are "required"